code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -270,7 +270,7 @@
 	id = "sand golem"
 	fixed_mut_color = "ffdc8f"
 	meat = /obj/item/weapon/ore/glass //this is sand
-	armor = 0
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = -100, "bio" = 100, "rad" = 100, "fire" = 0, "acid" = 0)
 	burnmod = 3 //melts easily
 	brutemod = 0.25
 	info_text = "As a <span class='danger'>Sand Golem</span>, you are immune to physical bullets and take very little brute damage, but are extremely vulnerable to burn damage. You will also turn to sand when dying, preventing any form of recovery."

```

Comment:
As a sand golem, you are immune to physical bullets and we're lying to you by saying you take very little brute when in reality you take full brute damage, fuck you